Dockerfile

```diff
@@ -281,6 +281,113 @@ COPY --from=rockylinux9-build /tmp/google-cloud-ops-agent.tgz /google-cloud-ops-
 COPY --from=rockylinux9-build /google-cloud-ops-agent*.rpm /
 COPY --from=rockylinux9-build /google-cloud-ops-agent-plugin*.tar.gz /
 
+# ======================================
+# Build Ops Agent for rockylinux-10
+# ======================================
+
+FROM rockylinux/rockylinux:10 AS rockylinux10-build-base
+ARG OPENJDK_MAJOR_VERSION
+
+RUN set -x; dnf -y update && \
+		dnf -y install 'dnf-command(config-manager)' && \
+		dnf config-manager --set-enabled crb && \
+		dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-10.noarch.rpm && \
+		dnf -y install git systemd \
+		autoconf libtool libcurl-devel libtool-ltdl-devel openssl-devel \
+		gcc gcc-c++ make cmake bison flex file systemd-devel zlib-devel gtest-devel rpm-build systemd-rpm-macros \
+		expect rpm-sign zip tzdata-java
+
+		ENV JAVA_HOME /usr/lib/jvm/java-${OPENJDK_MAJOR_VERSION}-openjdk/
+COPY --from=openjdk-install /usr/local/java-${OPENJDK_MAJOR_VERSION}-openjdk/ /usr/local/java-${OPENJDK_MAJOR_VERSION}-openjdk
```

Comment:
For posterity, shall we leave an explanation of why `yajl-devel` and the openjdk packages are excluded from this list? Either in a code comment or just here on the PR, either one should be discoverable enough

Reply:
Updated the PR description.